src/Internal/Plutip/Spawn.purs

```diff
@@ -88,3 +84,13 @@ killOnExit child = do
     alive <- Ref.read aliveRef
     when alive do
       kill SIGINT child
+
+foreign import _rmdirSync :: FilePath -> Effect Unit
+
+killOnExitAndRemDir :: ChildProcess -> FilePath -> FilePath -> Effect Unit
+killOnExitAndRemDir child workingDir testClusterDir = do
+  ChildProcess.onExit child \_ -> do
+    _rmdirSync workingDir
+  Process.onExit \_ -> do
+    _rmdirSync testClusterDir
+    _rmdirSync workingDir
```

Comment:
these effects don't do anything at the moment I think. at least they don't get triggered by a ctrl+c exit